igcollect/linux_disklatency.py

```diff
@@ -107,5 +77,46 @@ def main():
             print(template.format(mode, k, int(v)))
 
 
+def get_path(args) -> str:
+    if args.path is not None:
+        return args.path
+
+    # no --path is specified -> use libvirt to get the storage volume
+    from libvirt import (
+        open as libvirt_open,
+        libvirtError,
+    )
+
+    if args.storagepool is None or args.storagevol is None:
+        raise ArgumentError('if no --path is set you have to set --storagepool AND --storagevol')
```

Comment:
It feels a bit weird to have script argument validation inside a helper function. But, I can live with that for now. ^^